tests/pkg_tinyvcdiff/app.config.test

```diff
@@ -0,0 +1,6 @@
+CONFIG_MODULE_EMBUNIT=y
+CONFIG_PACKAGE_TINYVCDIFF=y
+CONFIG_MODULE_MTD=y
+CONFIG_PACKAGE_FATFS=y
+CONFIG_MODULE_FATFS_VFS=y
+CONFIG_MODULE_FATFS_VFS_FORMAT=y
```

Comment:
`FAT` does not seem to be yet moduled in Kconfig :(

Reply:
It got replaced by littlefs2 - so that problem has disappeared for my PR :D

Reply:
Well ... littlefs2 isn't, too.

I'll remove it from the `app.config.test`, right?

Reply:
Yes, vfs and filesystems are not yet modelled...

Reply:
b37095f26145fd45da9c83534b40e90716a4b719

Reply:
Okay, I see ... we cannot have just one half of the story. I have to model `littlfs2` in Kconfig before I can use Kconfig for this PR, right?